memoized_lcnaf.txt

```diff
@@ -1 +1 @@
d.loc.gov/authorities/names/n78086438-781", "United States of America"=>:absent, "Florida--Miami"=>"https://id.loc.gov/authorities/names/n81063248-781", "California--San Diego"=>"http://id.loc.gov/authorities/names/n79056560-781", "New York (State)--New York--Bronx"=>"https://id.loc.gov/authorities/names/n88074968-781", "Philippines"=>"http://id.loc.gov/authorities/names/n79058386", "Illinois Hospital Association"=>"http://id.loc.gov/authorities/names/n77015653", "United States. Child Nutrition and WIC Reauthorization Act of 2004"=>"http://id.loc.gov/authorities/names/no2004084135", "Cook County (Ill.). Department of Public Health"=>"http://id.loc.gov/authorities/names/no2010196651", "Illinois--Cook County"=>"https://id.loc.gov/authorities/names/n79060249-781", "Illinois--Chicago Region"=>:absent, "Armenia"=>"https://id.loc.gov/authorities/names/n79111504", "Cambodia"=>"http://id.loc.gov/authorities/names/n80014970", "Canada. Northwest Territories"=>"https://id.loc.gov/authorities/names/n84127936", "Caribbean Region"=>"https://id.loc.gov/authorities/subjects/sh85020279", "Caribbean Area"=>"https://id.loc.gov/authorities/subjects/sh85020279", "China"=>"https://id.loc.gov/authorities/names/n79091151", "Cyprus"=>"https://id.loc.gov/authorities/names/n79055857", "Czech Republic"=>"https://id.loc.gov/authorities/names/n81080131", "Estonia"=>"https://id.loc.gov/authorities/names/n82253985", "Faroe Islands"=>"https://id.loc.gov/authorities/names/n79035921", "Faeroe Islands"=>"https://id.loc.gov/authorities/names/n79035921", "Guam"=>"https://id.loc.gov/authorities/names/n50000810", "Illinois--Chicago Metropolitan Area"=>"https://id.loc.gov/authorities/subjects/sh2009008684-781", "Illinois--Chicago Region"=>"https://id.loc.gov/authorities/subjects/sh2009008684-781", "Illinois--Northern"=>"https://id.loc.gov/authorities/names/n84054350-781", "Ireland"=>"https://id.loc.gov/authorities/names/n79063445", "Isle of Mann"=>"https://id.loc.gov/authorities/names/n80106871", "Kelly, Florence"=>"http://id.loc.gov/authorities/names/n50060216", "Kelley, Florence, 1859-1932"=>"http://id.loc.gov/authorities/names/n50060216", "Laos"=>"https://id.loc.gov/authorities/names/n80128712", "Latin America"=>"http://id.loc.gov/authorities/subjects/sh85074879", "United Kingdom"=>"https://id.loc.gov/authorities/names/n79023147", "Great Britain"=>"https://id.loc.gov/authorities/names/n79023147", "Ukraine"=>"https://id.loc.gov/authorities/names/n81017756", "Moldova"=>"https://id.loc.gov/authorities/names/n90720192", "Lithuania"=>"https://id.loc.gov/authorities/names/n82209573", "Myanmar"=>"https://id.loc.gov/authorities/names/n80021691", "Burma"=>"https://id.loc.gov/authorities/names/n80021691", "New York (State)--Bronx County"=>"https://id.loc.gov/authorities/names/n88074968-781", "New York (State)--New York"=>"https://id.loc.gov/authorities/names/n79007751-781", "Serbia"=>"https://id.loc.gov/authorities/names/n85195919", "Slovakia"=>"https://id.loc.gov/authorities/names/n50075233"}
```

Comment:
It's difficult to look at, but what I have done here is replace the "absent" terms with the correct LCNAF links where I could and added new terms where they were missing. I tested on staging and locally to see if things were working okay and everything was still running correctly. Now it's just time to run another export.